src/sfizz/Config.h

```diff
@@ -38,6 +38,7 @@ namespace config {
     constexpr bool loggingEnabled { false };
     constexpr size_t numChannels { 2 };
     constexpr int numBackgroundThreads { 4 };
```

Comment:
Sample loading will be waiting while 4 are being processed?
Intuitively, it does not sound like much. How well does high load manage under this setting?

Reply:
This is probably what needs testing. It could be set relative to the CPU count.

Reply:
The threadpool now starts up `hardware_concurrency() - 2` threads, if available.